introduction.md

```diff
@@ -1,2 +1,4 @@
 My name is Ritchie and I love learning code.
-My favourite code school path is Javascript.
\ No newline at end of file
+My favourite code school path is Javascript.
+* javascript
+* git
\ No newline at end of file
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?